draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -312,13 +312,15 @@ Address A1    Address A2             Address B1    Address B2
       equally have been initiated from B1 or B2.
 
    *  The discovery and setup of additional subflows will be achieved
-      through a path management method;
+      through a path management method, whose definition is out of the
+      scope of this document.
```

Comment:
This document does specify how to add new subflows, so this text needs to be clarified. So it specifies how to add additional subflows, but not when (under what conditions). See the MPTCP RFC for comparison.